app/javascript/mastodon/features/account/components/header.jsx

```diff
@@ -438,7 +441,9 @@ class Header extends ImmutablePureComponent {
             <h1>
               <span dangerouslySetInnerHTML={displayNameHtml} />
               <small>
-                <span>@{acct}</span> {lockedIcon}
+                <span>@{username}</span>
```

Comment:
I wonder if the following change wouldn't prevent users accidentally copying usernames instead of the full handle:
```suggestion
                <span>@{username}<span class='invisible'>@{domain}</span></span>
```